src/Tribe/Freemius.php

```diff
@@ -15,8 +15,9 @@ class Tribe__Freemius {
 	private $instances = [];
 
 	/**
-	 * Loading of the vendor files for Freemius vendor
-	 * Freemius class should only be loaded once since it will be registred as a Singleton
+	 * Load the vendor files for Freemius vendor.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 26ee588d1042620ed631fb9b639a4c56</sub>